andela/AndelaEatsApi
# #160333550 paginate get all vendors route

#### What does this PR do?
>- Adds pagination functionality to `get all vendors` route
#### Description of Task to be completed?
>- refactor the endpoint to return paginated result, based on user input.
#### How should this be manually tested?
>- Run `ch-paginate-get-all-vendors-route-160333550:ch-paginate-get-all-vendors-route-160333550`, then run `git checkout ch-paginate-get-all-vendors-route-160333550`
> - run `purest` to ensure tests are passing
> - seed your database
> - Run `python run.py runserver`
> - make a get request to `api/v1/vendors`. You will get a paginated list of vendors.

#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- [#160333550](https://www.pivotaltracker.com/story/show/160333550)
#### Blocked Tickets

#### Screenshots (if appropriate)
>- N/A
![image](https://user-images.githubusercontent.com/24465059/45232330-f21c2680-b2c6-11e8-93c4-443ec65e3d73.png)

![image](https://user-images.githubusercontent.com/24465059/45232341-fea07f00-b2c6-11e8-8e46-31f29c57da22.png)

#### Questions:
>- N/A
